jujugui/static/gui/src/app/components/user-profile/test-user-profile.js

```diff
@@ -91,10 +91,62 @@ describe('UserProfile', () => {
               environmentCount={0}
               interactiveLogin={instance._interactiveLogin}
               username="test-owner" />
+            <div className="user-profile__empty twelve-col no-margin-bottom">
+              <img alt="Empty profile"
+                className="user-profile__empty-image"
+                src="/juju-ui/assets/images/non-sprites/empty_profile.png" />
```

Comment:
This image is not loading. I think it should be just in the `images` directory as there are only svg's in `non-sprite`